h/views/api_auth.py

```diff
@@ -31,6 +32,43 @@ def __init__(self, context, request):
     @view_config(request_method='GET',
                  renderer='h:templates/oauth/authorize.html.jinja2')
     def get(self):
+        return self._authorize()
+
+    @view_config(request_method='GET',
+                 request_param='response_mode=web_message',
+                 renderer='h:templates/oauth/authorize.html.jinja2')
+    def get_web_message(self):
```

Comment:
With this naming convention, I’m finding it easy to tell *when* each of these view methods gets invoked (for instance, when there’s a `GET` request with the `web_message` response mode), but I’m finding it hard to keep track of what the methods themselves are doing, particularly given that I’m a little hazy on what `_authorize` and `_authorized_response` are returning themselves). I don’t know whether the best solution to this is a different naming convention or a great big pile of docstrings, or whether it’s because I’m still a little unfamiliar with the details of the different flows.